tests/StackExchange.Redis.Tests/Failover.cs

```diff
@@ -281,10 +287,19 @@ public async Task SubscriptionsSurviveMasterSwitchAsync()
 
                     Assert.True(a.GetServer(TestConfig.Current.FailoverMasterServerAndPort).IsSlave, $"A Connection: {TestConfig.Current.FailoverMasterServerAndPort} should be a slave");
                     Assert.False(a.GetServer(TestConfig.Current.FailoverSlaveServerAndPort).IsSlave, $"A Connection: {TestConfig.Current.FailoverSlaveServerAndPort} should be a master");
+                    await UntilCondition(TimeSpan.FromSeconds(10), () => b.GetServer(TestConfig.Current.FailoverMasterServerAndPort).IsSlave).ForAwait();
                     var sanityCheck = b.GetServer(TestConfig.Current.FailoverMasterServerAndPort).IsSlave;
                     if (!sanityCheck)
                     {
-                        Skip.Inconclusive("Not enough latency.");
+                        Log("FAILURE: B has not detected the topology change.");
+                        foreach (var server in b.GetServerSnapshot().ToArray())
```

Comment:
why `ToArray()`? if this is because of spans+await, a: it'll *probably* work anyway, but if it doesn't (sometimes the compiler can, sometimes it can't), then suggest you create a static local method just for the log, that takes `b` and whatever `Log` needs (the `this` instance?)

Reply:
Yes indeed, await/span won't compile just went simple for test case!